src/assets/information-data/termsInformation.json

```diff
@@ -33,6 +33,11 @@
             "text": "Takstregler",
             "type": "link",
             "link": "https://www.atb.no/takstregler/"
+        },
+        {
+            "text": "Personvernerklæring",
+            "type": "link",
+            "link": "https://beta.atb.no/private-policy"
```

Comment:
All the urls in this file should probably come from firestore urls. In a new issue, maybe?

Reply:
Yeah. Can just do it in this one I guess.

Reply:
It turns out that this is something that should be fixed for all these:

- src/assets/information-data/termsInformation.json
- src/assets/information-data/ticketingInformation.json
- src/assets/information-data/ticketInspectionInformation.json 
- src/translations/screens/subscreens/Information.ts

So I think it should be a separate issue

Reply:
https://github.com/AtB-AS/kundevendt/issues/16282